autonomy/data/Dockerfiles/agent/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM valory/open-aea-user:latest
+
+ARG AEA_AGENT
+
+COPY openssl.cnf /etc/ssl
+COPY Pipfile .
+COPY . /home/ubuntu
+
+WORKDIR /home/ubuntu
+ENV PATH=$PATH:/home/ubuntu/.local/bin
+
+RUN sudo apt-get update --fix-missing
+RUN sudo apt-get autoremove
+RUN sudo apt-get autoclean
+RUN sudo apt-get install python3.10 python3.10-dev git -y
+RUN sudo apt-get install net-tools subversion -y
```

Comment:
why do we use subversion?

Reply:
We don't need this now, I'll remove this